extensions/functions_arithmetic.yaml

```diff
@@ -1614,3 +1614,131 @@ window_functions:
         decomposable: NONE
         return: i64?
         window_type: PARTITION
+  - name: "first_value"
+    description: >
+      The `expression` is evaluated against the first row.  The `expression` can be a
+      column, expression or subquery that evaluates to a single value.
+    impls:
+      - args:
+          - value: any1
+            name: expression
+        nullability: DECLARED_OUTPUT
+        decomposable: NONE
+        return: any1?
+        window_type: PARTITION
+  - name: "last_value"
+    description: >
+      The `expression` is evaluated against the last row.  The `expression` can be a
+      column, expression, or subquery that evaluates to a single value.
+    impls:
+      - args:
+          - value: any1
+            name: expression
+        nullability: DECLARED_OUTPUT
+        decomposable: NONE
+        return: any1?
+        window_type: PARTITION
+  - name: "nth_value"
+    description: >
+      The `expression` is evaluated against a row that comes after the current row based
```

Comment:
What happens if n is outside the range of the window?  E.g. a window of size 10 and n is 100.

Reply:
updated!  `null` is returned in this scenario